jsonrpc/filter_manager_test.go

```diff
@@ -292,6 +292,24 @@ func TestFilterTimeout(t *testing.T) {
 	assert.False(t, m.Exists(id))
 }
 
+func TestRemoveFilterByWebsocket(t *testing.T) {
+	store := newMockStore()
+
+	mock := &mockWsConn{
+		msgCh: make(chan []byte, 1),
+	}
+
+	m := NewFilterManager(hclog.NewNullLogger(), store)
+	go m.Run()
```

Comment:
Please close after the test

Reply:
Fixed:

[246f04e](https://github.com/0xPolygon/polygon-edge/pull/570/commits/246f04eeb3f24e18b3807fe59dd54343f7fc7441)